java/ql/test/experimental/query-tests/security/CWE-555/context.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Context>
+    <!-- BAD: Password of datasource is not encrypted -->
+    <Resource name="jdbc/exampleDS1" auth="Container" type="javax.sql.DataSource" maxTotal="100" maxIdle="30" maxWaitMillis="10000" username="root" password="1234" driverClassName="com.mysql.jdbc.Driver" url="jdbc:mysql://www.example1.com:3306/proj" />
```

Comment:
Maybe it would be good to revert the changes to this file. The formatting here was fine, and in my opinion it is easier to read when the attributes are distributed over multiple lines.

Reply:
Agreed. I've reverted the changes for better readability.